server/src/main/java/org/apache/iotdb/db/exception/metadata/RootNotExistException.java

```diff
@@ -15,16 +15,16 @@
  * KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations
  * under the License.
+ *
  */
 
-package org.apache.iotdb.db.exception.storageGroup;
+package org.apache.iotdb.db.exception.metadata;
 
-public class StorageGroupPathException extends StorageGroupException {
+public class RootNotExistException extends MetadataException {
```

Comment:
This method used to call `StorageGroupPathException` because it shows the storage group doesn't exist, but in the **cluster** branch, it is renamed to be thrown when the `root` node is wrong. For example when users enter `rooot.vehicle.d0`, he will get this error.

Reply:
Incorrect root exception

Reply:
Renamed